src/python/pants/backend/javascript/dependency_inference/rules.py

```diff
@@ -256,7 +249,11 @@ async def infer_js_source_dependencies(
     _handle_unowned_imports(
         request.field_set.address,
         nodejs_infer.unowned_dependency_behavior,
-        frozenset(string for string, addresses in imports.items() if not addresses),
+        frozenset(
+            string
+            for string, addresses in imports.items()
+            if not addresses and not string.startswith("node:")
```

Comment:
Where's the `node:` come from and why are they ignored? Is that worth a comment?

Reply:
Extracted a documenting method and a link to the node.js spec